test-single-read/nextflow.config

```diff
@@ -0,0 +1,37 @@
+/************************************************
+| CONFIGURATION FILE FOR NAO VIRAL MGS WORKFLOW |
+************************************************/
+
+params {
+    mode = "run_dev_se"
+
+
+    // Directories
+    base_dir = "s3://nao-mgs-simon/test_single_read" // Parent for working and output directories (can be S3)
+    ref_dir = "s3://nao-mgs-wb/index-20241113/output" // Reference/index directory (generated by index workflow)
+
+    // Files
+    sample_sheet = "${launchDir}/samplesheet.csv" // Path to library TSV
+    adapters = "${projectDir}/ref/adapters.fasta" // Path to adapter file for adapter trimming
+
+    // Whether the underlying data is paired-end or single-end
+    single_end = params.sample_sheet.toString().readLines()[0].contains('fastq_2') ? false : true
```

Comment:
Defining single_end should probably go in the workflow itself. Also why are there two configs that are different 'configs/run_dev_se.config'.

Reply:
Done